lib/qmp.ml

```diff
@@ -65,6 +67,8 @@ type command =
   | Add_fd of int option
   | Remove_fd of int
   | Blockdev_change_medium of string * string
+  | Device_del of string
```

Comment:
Does qemu support also hot plug? If that is the case, I'd like both plug and unplug being added by this PR, even if we do not plan to use the hotplug.

Reply:
Yes, adding it in a separate commit of this PR is perfectly fine